code/drasil-docLang/package.yaml

```diff
@@ -20,14 +20,15 @@ dependencies:
 - pretty
 - transformers
 - multiplate
-- drasil-lang
+- drasil-code
```

Comment:
This is needed exclusively because of `programName`!!!!

```haskell
-- | Convert an abbreviation into one deemed 'code-friendly', removing spaces,
--   and replacing special characters with underscores.
--
--   FIXME: This should NOT be treated as a 'getter', but something we cache
--   local to something that has a 'program name'.
programName :: CommonIdea c => c -> String
programName = toPlainName . filter (not . isSpace) . abrv
```